micro-rdk-server/esp32/ota_8mb_partitions.csv

```diff
@@ -1,7 +1,8 @@
 # Name,   Type, SubType, Offset,  Size, Flags
 # Note: if you have increased the bootloader size, make sure to update the offsets to avoid overlap
-nvs,	       data,	nvs,	  0x9000,	0x6000,
-otadata,       data,	ota,	  0xF000,	0x2000,
-phy_init,      data,	phy, 	  0x11000, 	0x1000,
-ota_0,	       0,	ota_0,	  ,		0x377000,
-ota_1,	       0,	ota_1,	  ,		0x377000,
+nvs,         data,  nvs,      0x9000,   0x6000,
+otadata,     data,  ota,      0xF000,   0x2000,
+phy_init,    data,  phy,      0x11000,  0x1000,
+ota_0,       app,   ota_0,    0x20000,  3968K,
```

Comment:
56KB lost to get 0x10000 alignment requirement for app partitions

Reply:
> Meanwhile, I'm not even entirely sure we are actually using the phy init, since I don't think we have `CONFIG_ESP_PHY_INIT_DATA_IN_PARTITION` set?

Oh, you're right. It's [off by default](https://docs.espressif.com/projects/esp-idf/en/stable/esp32/api-reference/kconfig.html?highlight=config_esp_phy_init_data_in_partition#config-esp-phy-init-data-in-partition). I don't think I've looked into `phy_init` before. it's always been in the examples.

We can look into it further. If we truly aren't using it we can:
a) allocate it more to NVS or app partitions
b) keep it as reserved 
c) keep it there so that if/when someone starts using it

Reply:
per last meeting, leaving it as-is